test/sqllogictest/topk.slt

```diff
@@ -54,29 +54,39 @@ EXPLAIN PLAN FOR SELECT state, name FROM
 ----
 %0 =
 | Get materialize.public.cities (u1)
-| Distinct group=(#1)
+| TopK group=(#1) order=(#2 desc) limit=3 offset=0
+| Project (#1, #0)
+
+EOF
+
+query T multiline
+EXPLAIN PLAN FOR SELECT state, name FROM
+    (SELECT DISTINCT state FROM cities) grp
+    LEFT JOIN LATERAL (SELECT name, pop FROM cities  where cities.state = grp.state ORDER BY pop DESC LIMIT 3) ON state = grp.state
```

Comment:
The `ON` clause in this join looks like it might still be tautological?

Reply:
We discussed this, and to recap: it seems like we want `true` as the join condition to test left joins as well, is that right? This still requires the additional logic that may introduce nulls, but the joins should still be optimized out (the thing this tests for).

Reply:
That sounds right to me!